applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py

```diff
@@ -29,14 +26,17 @@ def ConvectionVelocity(x, y, z):
     vel[0] = 1.0
     return vel
 
+def GetFilePath(fileName): 
+    return os.path.join(pathlib.Path(__file__).absolute().parent, fileName)

```

Comment:
```suggestion
    return pathlib.Path(__file__).absolute().parent / fileName

```

the goal is to get rid of `os.path` altogether

Reply:
I tried this and didn't work, let me a second and I post the error

Reply:
Okay, it is because it not a string, wait a sec

Reply:
Now